In [1]:
from qutip import *
import numpy as np
from scipy.linalg import block_diag

def eigenvectmatr(M):
    a=np.empty((M.shape[0], 0))
    for i in range(M.shape[0]):
        a = np.append(a,np.array(M.eigenstates()[1][i])) 
    return Qobj(a.reshape((M.shape[0],M.shape[0]))).dag()

Take 3 particles with $s=\frac{1}{2}$.

## Transformation A: $\langle \alpha\alpha\alpha\vert T_+\alpha\rangle$

("Partly") Coupled: $\vert T_+ \alpha \rangle, \vert T_0 \alpha \rangle, \vert S \alpha \rangle,\vert T_- \alpha \rangle, \vert T_+ \beta \rangle, \vert T_0 \beta \rangle, \vert S \beta \rangle,\vert T_- \beta \rangle$

Uncoupled: $\langle \alpha \alpha \alpha \vert, \langle \alpha \beta \alpha \vert,  \langle \beta \alpha \alpha \vert, \langle \beta \beta \alpha \vert, \langle \alpha \alpha \beta \vert,\langle \alpha \beta \beta \vert,\langle \beta \alpha \beta \vert,\langle \beta \beta \beta \vert$

Transformation matrix: $T_A$

In [2]:
T_2x2 = [[1,0,0,0],[0,1/np.sqrt(2),1/np.sqrt(2),0],[0,1/np.sqrt(2),-1/np.sqrt(2),0],[0,0,0,1]]
T_A = Qobj(block_diag(*([T_2x2] * 2)))
T_A

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = True
Qobj data =
[[ 1.          0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.          0.70710678  0.70710678  0.          0.          0.
   0.          0.        ]
 [ 0.          0.70710678 -0.70710678  0.          0.          0.
   0.          0.        ]
 [ 0.          0.          0.          1.          0.          0.
   0.          0.        ]
 [ 0.          0.          0.          0.          1.          0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.70710678
   0.70710678  0.        ]
 [ 0.          0.          0.          0.          0.          0.70710678
  -0.70710678  0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.          1.        ]]

## Transformation B: $\langle T_+\alpha\vert \frac{3}{2},\frac{3}{2}\rangle$

[Adapted from Binney & Cohen](https://arxiv.org/abs/quant-ph/0605180) 

$j_1=1$ and $j_2=\frac{1}{2}$ 

$\mathbf{3\otimes2=4\oplus2}$  , so need 6x6 matrix

Total angular momentum: $j=\{\frac{3}{2},\frac{1}{2} \}$

Coupled states: $\vert j,m\rangle \quad$ 

Uncoupled states: basis of "partly" coupled states with 2 spins coupled and 1 uncoupled 

Angular momentum coupled sequentially: first took $\vec{j}_p+\vec{j}_q=\vec{j}_{pq}$ and now taking $\vec{j}_{pq}+\vec{j}_r=\vec{j}_{pqr}$

Start with stretched state $\vert\frac{3}{2},\frac{3}{2}\rangle$ and apply $J_-$ sequentially to get 4 states with $j=\frac{3}{2}$

$\vert\frac{3}{2},\frac{3}{2}\rangle = \vert\alpha\rangle\vert 1,1\rangle = \vert T_+ \alpha\rangle$ 

$\vert\frac{3}{2},\frac{1}{2}\rangle = \sqrt{\frac{1}{3}}\vert \beta\rangle\vert 1,1\rangle + \sqrt{\frac{2}{3}}\vert \alpha\rangle\vert 1,0\rangle  = \sqrt{\frac{1}{3}}\vert T_+ \beta\rangle+\sqrt{\frac{2}{3}}\vert T_0 \alpha\rangle$ 

$\vert\frac{3}{2},-\frac{1}{2}\rangle = \sqrt{\frac{2}{3}}\vert \beta\rangle\vert 1,0\rangle + \sqrt{\frac{1}{3}}\vert \alpha\rangle\vert 1,-1\rangle  = \sqrt{\frac{2}{3}}\vert T_0 \beta\rangle+\sqrt{\frac{1}{3}}\vert T_- \alpha\rangle$ 

$\vert\frac{3}{2},-\frac{3}{2}\rangle = \vert \beta\rangle\vert 1,-1\rangle = \vert T_- \beta\rangle$ 

To get $j=\frac{1}{2}$ states construct orthogonal linear combinations to states with the same $m$:

$\vert\frac{1}{2},\frac{1}{2}\rangle = \sqrt{\frac{2}{3}}\vert \beta\rangle\vert 1,1\rangle - \sqrt{\frac{1}{3}}\vert \alpha\rangle\vert 1,0\rangle  = \sqrt{\frac{2}{3}}\vert T_+ \beta\rangle-\sqrt{\frac{1}{3}}\vert T_0 \alpha\rangle$

$\vert\frac{1}{2},-\frac{1}{2}\rangle = \sqrt{\frac{1}{3}}\vert \beta\rangle\vert 1,0\rangle - \sqrt{\frac{2}{3}}\vert \alpha\rangle\vert 1,-1\rangle  = \sqrt{\frac{1}{3}}\vert T_0 \beta\rangle-\sqrt{\frac{2}{3}}\vert T_- \alpha\rangle$ 


Works when only 3 uncoupled two-spin states possible: $\vert T_+\rangle=(1, 0, 0)$; $\vert T_0\rangle=(0, 1, 0)$ and $\vert T_-\rangle=(0, 0, 1)$

Need "coupling" of $j=0$ to $j=\frac{1}{2}$: see below, taking those states as is: $j_a+0=j_a$

Construct 6x6 matrix T_B6:

In [3]:
T_B6 = Qobj([[1,0,0,0,0,0],[0,np.sqrt(1/3),0,0,np.sqrt(2/3),0],[0,np.sqrt(2/3),0,0,-np.sqrt(1/3),0],[0,0,np.sqrt(2/3),0,0,np.sqrt(1/3)],[0,0,np.sqrt(1/3),0,0,-np.sqrt(2/3)],[0,0,0,1,0,0]])
T_B6

Quantum object: dims = [[6], [6]], shape = (6, 6), type = oper, isherm = False
Qobj data =
[[ 1.          0.          0.          0.          0.          0.        ]
 [ 0.          0.57735027  0.          0.          0.81649658  0.        ]
 [ 0.          0.81649658  0.          0.         -0.57735027  0.        ]
 [ 0.          0.          0.81649658  0.          0.          0.57735027]
 [ 0.          0.          0.57735027  0.          0.         -0.81649658]
 [ 0.          0.          0.          1.          0.          0.        ]]

## Transformation C: $\langle \alpha\alpha\alpha\vert \frac{3}{2},\frac{3}{2}\rangle$

After [Paris](https://public.lanl.gov/mparis/qmp.pdf) with significant corrections, and [Magnasco](https://www.sciencedirect.com/science/article/pii/B9780444626479000099?via%3Dihub)

$N=3 \;\rightarrow\; 2^N=8 \;\rightarrow\; \mathbf{4\otimes2=4\oplus2\oplus2}$ , so need 8x8 matrix and get 1 quartet $j=\frac{3}{2}$ and 2 doublets $j=\frac{1}{2}$.

$\vert j, m; j_{12}\rangle$:

Again, construct first 4 from stretched state and lowering operator:

$\vert \frac{3}{2}, +\frac{3}{2};1\rangle=\vert\alpha\alpha\alpha\rangle$

$\vert \frac{3}{2}, +\frac{1}{2};1\rangle=\frac{1}{\sqrt{3}}\Big(\vert\alpha\alpha\beta\rangle+\vert\alpha\beta\alpha\rangle+\vert\beta\alpha\alpha\rangle\Big)$

$\vert \frac{3}{2}, -\frac{1}{2};1\rangle=\frac{1}{\sqrt{3}}\Big(\vert\alpha\beta\beta\rangle+\vert\beta\alpha\beta\rangle+\vert\beta\beta\alpha\rangle\Big)$

$\vert \frac{3}{2}, -\frac{3}{2};1\rangle=\vert\beta\beta\beta\rangle$

Then 2 orthogonal / from Clebsh-Gordan:

$\vert \frac{1}{2}, +\frac{1}{2};1\rangle= \sqrt{\frac{2}{3}}\vert 1,+1\rangle\otimes\vert\beta\rangle-\sqrt{\frac{1}{3}}\vert 1,0\rangle\otimes\vert\alpha\rangle=\frac{1}{\sqrt{6}}\Big(2\vert\alpha\alpha\beta\rangle - \vert\alpha\beta\alpha\rangle - \vert\beta\alpha\alpha\rangle     \Big)$

$\vert \frac{1}{2}, -\frac{1}{2};1\rangle= \sqrt{\frac{1}{3}}\vert 1,0\rangle\otimes\vert\beta\rangle-\sqrt{\frac{2}{3}}\vert 1,-1\rangle\otimes\vert\alpha\rangle=\frac{1}{\sqrt{6}}\Big(\vert\alpha\beta\beta\rangle + \vert\beta\alpha\beta\rangle - 2\vert\beta\beta\alpha\rangle     \Big)$

And 2 directly from $\vert 0,0\rangle$:

$\vert \frac{1}{2}, +\frac{1}{2};0\rangle=\vert 0,0\rangle\otimes\vert\alpha\rangle=\frac{1}{\sqrt{2}}\Big(\vert\alpha\beta\alpha\rangle-\vert\beta\alpha\alpha\rangle\Big)$

$\vert \frac{1}{2}, -\frac{1}{2};0\rangle=\vert 0,0\rangle\otimes\vert\beta\rangle=\frac{1}{\sqrt{2}}\Big(\vert\alpha\beta\beta\rangle-\vert\beta\alpha\beta\rangle\Big)$



In [4]:
T_C = np.matrix([[1,0,0,0,0,0,0,0],
      [0,1/np.sqrt(3),0,0,-1/np.sqrt(6),0,-1/np.sqrt(2),0],
      [0,1/np.sqrt(3),0,0,-1/np.sqrt(6),0,1/np.sqrt(2),0],
      [0,0,1/np.sqrt(3),0,0,-2/np.sqrt(6),0,0],
      [0,1/np.sqrt(3),0,0,2/np.sqrt(6),0,0,0],
      [0,0,1/np.sqrt(3),0,0,1/np.sqrt(6),0,-1/np.sqrt(2)],
      [0,0,1/np.sqrt(3),0,0,1/np.sqrt(6),0,1/np.sqrt(2)],
      [0,0,0,1,0,0,0,0]])
T_C = Qobj(T_C)
T_C

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 1.          0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.          0.57735027  0.          0.         -0.40824829  0.
  -0.70710678  0.        ]
 [ 0.          0.57735027  0.          0.         -0.40824829  0.
   0.70710678  0.        ]
 [ 0.          0.          0.57735027  0.          0.         -0.81649658
   0.          0.        ]
 [ 0.          0.57735027  0.          0.          0.81649658  0.
   0.          0.        ]
 [ 0.          0.          0.57735027  0.          0.          0.40824829
   0.         -0.70710678]
 [ 0.          0.          0.57735027  0.          0.          0.40824829
   0.          0.70710678]
 [ 0.          0.          0.          1.          0.          0.
   0.          0.        ]]

### To verify / find 2 missing states in Transformation B - take matrix products:

###  $\bbox[5px,border:2px solid red]{\langle \alpha\alpha\alpha\vert T_+\alpha\rangle_A \cdot \langle T_+\alpha\vert \frac{3}{2},\frac{3}{2}\rangle_B = \langle \alpha\alpha\alpha\vert \frac{3}{2},\frac{3}{2}\rangle_C}$

$T_A T_B = T_C$

$T_B = T^{-1}_A T_C$

In [5]:
T_B = T_A.inv()*T_C
T_B                      

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 1.          0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.          0.81649658  0.          0.         -0.57735027  0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
  -1.          0.        ]
 [ 0.          0.          0.57735027  0.          0.         -0.81649658
   0.          0.        ]
 [ 0.          0.57735027  0.          0.          0.81649658  0.
   0.          0.        ]
 [ 0.          0.          0.81649658  0.          0.          0.57735027
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.         -1.        ]
 [ 0.          0.          0.          1.          0.          0.
   0.          0.        ]]

In [6]:
T_B6

Quantum object: dims = [[6], [6]], shape = (6, 6), type = oper, isherm = False
Qobj data =
[[ 1.          0.          0.          0.          0.          0.        ]
 [ 0.          0.57735027  0.          0.          0.81649658  0.        ]
 [ 0.          0.81649658  0.          0.         -0.57735027  0.        ]
 [ 0.          0.          0.81649658  0.          0.          0.57735027]
 [ 0.          0.          0.57735027  0.          0.         -0.81649658]
 [ 0.          0.          0.          1.          0.          0.        ]]

Therefore $\vert \frac{1}{2},\frac{1}{2},0\rangle=-\vert S_0\alpha\rangle$ and $\vert \frac{1}{2},-\frac{1}{2},0\rangle=-\vert S_0\beta\rangle$

<span style="color:red">TO DO:</span> Significance of minus sign? (normalisation gets rid of it...)

---
---

# Transforming the Hamiltonian

Consider system of three spin-1/2 particles made up of two electrons and a nuclear spin. 

The Hamiltonian for the hyperfine interaction between one electron spin and the nuclear spin:

$\mathbf{H} = \mathbf{S_A S_C} = \mathbf{S_{Ax} S_{Cx}} + \mathbf{S_{Ay}S_{Cy}} + \mathbf{S_{Az}S_{Cz}}$


### $\mathbf{H_c = T^\dagger H T}$


---

### Uncoupled:

In [7]:
idt = qeye(2)
sax, sbx, scx = tensor(sigmax()/2,idt,idt), tensor(idt,sigmax()/2,idt),tensor(idt,idt,sigmax()/2)
say, sby, scy = tensor(sigmay()/2,idt,idt), tensor(idt,sigmay()/2,idt),tensor(idt,idt,sigmay()/2)
saz, sbz, scz = tensor(sigmaz()/2,idt,idt), tensor(idt,sigmaz()/2,idt),tensor(idt,idt,sigmaz()/2)

H = (sax*scx + say*scy + saz*scz)
print(H.eigenenergies())
H

[-0.75 -0.75  0.25  0.25  0.25  0.25  0.25  0.25]


Quantum object: dims = [[2, 2, 2], [2, 2, 2]], shape = (8, 8), type = oper, isherm = True
Qobj data =
[[ 0.25  0.    0.    0.    0.    0.    0.    0.  ]
 [ 0.   -0.25  0.    0.    0.5   0.    0.    0.  ]
 [ 0.    0.    0.25  0.    0.    0.    0.    0.  ]
 [ 0.    0.    0.   -0.25  0.    0.    0.5   0.  ]
 [ 0.    0.5   0.    0.   -0.25  0.    0.    0.  ]
 [ 0.    0.    0.    0.    0.    0.25  0.    0.  ]
 [ 0.    0.    0.    0.5   0.    0.   -0.25  0.  ]
 [ 0.    0.    0.    0.    0.    0.    0.    0.25]]

In [8]:
eigenvectmatr(H)

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 0.          0.          0.          1.          0.          0.
   0.          0.        ]
 [ 0.70710678  0.         -0.70710678  0.          0.          0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   1.          0.        ]
 [ 0.          0.70710678  0.          0.         -0.70710678  0.
   0.          0.        ]
 [-0.70710678  0.         -0.70710678  0.          0.          0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          1.
   0.          0.        ]
 [ 0.         -0.70710678  0.          0.         -0.70710678  0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.          1.        ]]

### A: Transforms to parly coupled $\vert T_+,\alpha\rangle$ representation

In [9]:
H_A=H.transform(T_A)
print(H_A.eigenenergies())
H_A

[-0.75 -0.75  0.25  0.25  0.25  0.25  0.25  0.25]


Quantum object: dims = [[2, 2, 2], [2, 2, 2]], shape = (8, 8), type = oper, isherm = True
Qobj data =
[[ 0.25        0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.          0.         -0.25        0.          0.35355339  0.
   0.          0.        ]
 [ 0.         -0.25        0.          0.          0.35355339  0.
   0.          0.        ]
 [ 0.          0.          0.         -0.25        0.          0.35355339
  -0.35355339  0.        ]
 [ 0.          0.35355339  0.35355339  0.         -0.25        0.
   0.          0.        ]
 [ 0.          0.          0.          0.35355339  0.          0.
   0.25        0.        ]
 [ 0.          0.          0.         -0.35355339  0.          0.25
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.          0.25      ]]

In [10]:
eigenvectmatr(H_A)

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 0.          0.          0.          0.          0.          1.
   0.          0.        ]
 [ 0.         -0.5         0.8660254   0.          0.          0.
   0.          0.        ]
 [ 0.         -0.5        -0.28867513 -0.81649658  0.          0.
   0.          0.        ]
 [-0.70710678  0.          0.          0.          0.          0.
   0.         -0.70710678]
 [ 0.          0.70710678  0.40824829 -0.57735027  0.          0.
   0.          0.        ]
 [ 0.5         0.          0.          0.          0.70710678  0.
   0.         -0.5       ]
 [-0.5         0.          0.          0.          0.70710678  0.
   0.          0.5       ]
 [ 0.          0.          0.          0.          0.          0.
   1.          0.        ]]

### C: Transforms to fully coupled $\vert j,m\rangle$ representation

In [11]:
H_C=H.transform(T_C)
print(H_C.eigenenergies())
H_C

[-0.75 -0.75  0.25  0.25  0.25  0.25  0.25  0.25]


Quantum object: dims = [[2, 2, 2], [2, 2, 2]], shape = (8, 8), type = oper, isherm = True
Qobj data =
[[ 0.25        0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.         -0.48570226 -0.23570226  0.          0.11785113  0.
   0.         -0.35355339]
 [ 0.         -0.23570226 -0.48570226  0.          0.11785113  0.
   0.          0.35355339]
 [ 0.          0.          0.          0.25        0.          0.
   0.          0.        ]
 [ 0.          0.11785113  0.11785113  0.          0.22140452  0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.25
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.25        0.        ]
 [ 0.         -0.35355339  0.35355339  0.          0.          0.
   0.         -0.25      ]]

In [12]:
eigenvectmatr(H_C)

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 0.          0.          0.          0.          0.          1.
   0.          0.        ]
 [ 0.85773088  0.          0.51409896  0.          0.          0.
   0.          0.        ]
 [ 0.27479745 -0.81251992 -0.45847644  0.          0.          0.
  -0.23258782  0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.          1.        ]
 [-0.13739873  0.0985752   0.22923822  0.          0.          0.
  -0.95856891  0.        ]
 [ 0.          0.          0.          0.          1.          0.
   0.          0.        ]
 [ 0.          0.          0.          1.          0.          0.
   0.          0.        ]
 [ 0.41219618  0.57453835 -0.68771466  0.          0.          0.
  -0.16446442  0.        ]]

### B: This doesn't match the bras of H 

In [13]:
H_B=H.transform(T_B)
print(H_B.eigenenergies())
H_B

[-0.75 -0.75  0.25  0.25  0.25  0.25  0.25  0.25]


Quantum object: dims = [[2, 2, 2], [2, 2, 2]], shape = (8, 8), type = oper, isherm = True
Qobj data =
[[ 0.25        0.          0.          0.          0.          0.
   0.          0.        ]
 [ 0.         -0.72140452  0.          0.          0.16666667  0.
   0.          0.        ]
 [ 0.          0.         -0.25        0.          0.          0.
   0.         -0.5       ]
 [ 0.          0.          0.          0.25        0.          0.
   0.          0.        ]
 [ 0.          0.16666667  0.          0.          0.22140452  0.
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.25
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   0.25        0.        ]
 [ 0.          0.         -0.5         0.          0.          0.
   0.         -0.25      ]]

In [14]:
eigenvectmatr(H_B)

Quantum object: dims = [[8], [8]], shape = (8, 8), type = oper, isherm = False
Qobj data =
[[ 0.          0.          1.          0.          0.          0.
   0.          0.        ]
 [ 0.98559856  0.          0.          0.          0.          0.
   0.         -0.16910198]
 [ 0.         -0.70710678  0.          0.          0.          0.70710678
   0.          0.        ]
 [ 0.          0.          0.          0.          0.          0.
   1.          0.        ]
 [-0.16910198  0.          0.          0.          0.          0.
   0.         -0.98559856]
 [ 0.          0.          0.          0.         -1.          0.
   0.          0.        ]
 [ 0.          0.          0.          1.          0.          0.
   0.          0.        ]
 [ 0.         -0.70710678  0.          0.          0.         -0.70710678
   0.          0.        ]]

-----

## Eigenstates:

<span style="color:red">TO DO:</span> Check if correct!

$\begin{array} {|r|r|}
\hline Energy & Uncoupled \, repr. & Partly \, coupled \, repr. & Coupled \, repr. \\ 
\hline E_1 =-\frac{3}{4} & \frac{1}{\sqrt{2}}\vert\alpha\beta\alpha\rangle-\frac{1}{\sqrt{2}}\vert\alpha\alpha\beta\rangle & +\frac{1}{2}\vert T_0\beta\rangle-\frac{1}{2}\vert S_0\beta\rangle-\frac{1}{\sqrt{2}}\vert T_-\alpha\rangle & 0.86 \vert\frac{3}{2},+\frac{1}{2},1\rangle +0.28\vert\frac{3}{2},-\frac{1}{2},1\rangle -0.14 \vert\frac{1}{2},+\frac{1}{2},1\rangle +0.41 \vert\frac{1}{2},-\frac{1}{2},0\rangle \\ 
\hline E_2 =-\frac{3}{4} & \frac{1}{\sqrt{2}}\vert\beta\beta\alpha\rangle-\frac{1}{\sqrt{2}}\vert\beta\alpha\beta\rangle & -\frac{1}{2}\vert T_0\alpha\rangle-\frac{1}{2}\vert S_0\alpha\rangle+\frac{1}{\sqrt{2}}\vert T_+\beta\rangle & -0.81\vert\frac{3}{2},-\frac{1}{2},1\rangle +0.10 \vert\frac{1}{2},+\frac{1}{2},1\rangle +0.58 \vert\frac{1}{2},-\frac{1}{2},0\rangle \\ 
\hline E_3 =+\frac{1}{4} & -\frac{1}{\sqrt{2}}\vert\alpha\beta\alpha\rangle-\frac{1}{\sqrt{2}}\vert\alpha\alpha\beta\rangle & \frac{\sqrt{3}}{2}\vert T_0\alpha\rangle-\frac{\sqrt{3}}{6}\vert S_0\alpha\rangle+\frac{1}{\sqrt{6}}\vert T_+\beta\rangle & 0.52 \vert\frac{3}{2},+\frac{1}{2},1\rangle -0.46\vert\frac{3}{2},-\frac{1}{2},1\rangle -+0.22 \vert\frac{1}{2},+\frac{1}{2},1\rangle -0.69 \vert\frac{1}{2},-\frac{1}{2},0\rangle \\ 
\hline E_4 =+\frac{1}{4} & -\frac{1}{\sqrt{2}}\vert\beta\beta\alpha\rangle-\frac{1}{\sqrt{2}}\vert\beta\alpha\beta\rangle &  -\frac{1}{\sqrt{2}}\vert T_-\alpha\rangle-\frac{1}{2}\vert T_0\beta\rangle+\frac{1}{2}\vert S_0\beta\rangle &  -0.23\vert\frac{3}{2},-\frac{1}{2},1\rangle -0.96 \vert\frac{1}{2},+\frac{1}{2},1\rangle -0.16 \vert\frac{1}{2},-\frac{1}{2},0\rangle\\ 
\hline E_5 =+\frac{1}{4} & \vert\beta\alpha\alpha\rangle & \frac{1}{\sqrt{2}}\vert T_0\beta\rangle+\frac{1}{\sqrt{2}}\vert S_0\beta\rangle &  \vert\frac{1}{2},+\frac{1}{2},0\rangle\\ 
\hline E_6 =+\frac{1}{4} & \vert\alpha\beta\beta\rangle & -\frac{2}{\sqrt{6}}\vert S_0\alpha\rangle-\frac{1}{\sqrt{3}}\vert T_+\beta\rangle& \vert\frac{1}{2},-\frac{1}{2},1\rangle\\ 
\hline E_7 =+\frac{1}{4} &\vert\alpha\alpha\alpha\rangle & \vert T_+\alpha\rangle &  \vert\frac{3}{2},+\frac{3}{2},1\rangle \\ 
\hline E_8 =+\frac{1}{4} & \vert\beta\beta\beta\rangle & \vert T_-\beta\rangle & \vert\frac{3}{2},-\frac{3}{2},1\rangle \\ 
\hline  \end{array}$